src/components/ui/AlertsSection/index.tsx

```diff
@@ -1,27 +1,33 @@
 import React from 'react';
 import PageSection from '../PageSection';
 import { AlertGroup, AlertActionCloseButton, Alert } from '@patternfly/react-core';
-import { AlertProps } from '../../../features/alerts/alertsSlice';
+import { AlertProps, removeAlert } from '../../../features/alerts/alertsSlice';
 
 import './AlertsSection.css';
+import { AlertsContext } from '../../AlertsContextProvider';
 
-type AlertsSectionProps = {
-  alerts: AlertProps[];
-  onClose: (alert: AlertProps) => void;
-};
+const AlertsSection: React.FC = () => {
+  const { alerts, dispatchAlertsAction } = React.useContext(AlertsContext);
+  const onClose = (alert: AlertProps) => dispatchAlertsAction(removeAlert(alert.key));
 
-const AlertsSection: React.FC<AlertsSectionProps> = ({ alerts, onClose }) =>
-  alerts.length ? (
-    <PageSection padding={{ default: 'noPadding' }}>
-      <AlertGroup className="alerts-section">
-        {alerts.map((alert) => (
-          // eslint-disable-next-line react/jsx-key
-          <Alert actionClose={<AlertActionCloseButton onClose={() => onClose(alert)} />} {...alert}>
-            {alert.message}
-          </Alert>
-        ))}
-      </AlertGroup>
-    </PageSection>
-  ) : null;
+  if (alerts.length) {
+    return (
+      <PageSection padding={{ default: 'noPadding' }}>
+        <AlertGroup className="alerts-section">
+          {alerts.map((alert) => (
+            // eslint-disable-next-line react/jsx-key
```

Comment:
use `alert.key` field as key ?

Reply:
`{...alert}` below adds the key correctly
![image](https://user-images.githubusercontent.com/1121740/86926567-8610e080-c132-11ea-8dc6-aad14f210ce2.png)